### PR TITLE
doc: proofread changelogs/unreleased for 2.10.0

### PR DESCRIPTION
**Pull request by xuniq**
Related to https://github.com/tarantool/doc/issues/2868

### CODE REVIEW

File: changelogs/unreleased/gh-7076-tz-attributes.md [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
-datetime values:
+  Timezone abbreviations can be used in addition to the timezone offset.
+  They can be used during constructing or modifying a date object, or while
+  parsing datetime literals. Numeric time offsets and named abbreviations
+  produce equivalent datetime values:
 
-```
-local date = require('datetime')
-local d2 = date.parse('2000-01-01T02:00:00 MSK')
+  ```
```
[the remark this answers is not in the document]
Comment (xuniq) [reply changelogs/unreleased/gh-7076-tz-attributes.md:12]: Sounds good. I'll update this code block.

File: changelogs/unreleased/gh-7129-improve-check-for-dangerous-select-calls.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ## feature/box
 
-* Improved check for dangerous select calls: calls with `offset + limit <= 100`
-  are now considered safe (i.e., a warning is not issued); 'ALL', 'GE', 'GT',
-  'LE', 'LT' iterators are now considered dangerous by default even with key
-  present (gh-7129).
+* Improved checking for dangerous `select` calls. The calls with
+  `offset + limit <= 100` are now considered safe, which means a warning is not
```
Comment (Totktonada) [inline changelogs/unreleased/gh-7129-improve-check-for-dangerous-select-calls.md:4]: 1000 should be here, it is the typo. See https://github.com/tarantool/tarantool/pull/7131#discussion_r875926566.

**Review by Totktonada — APPROVED** [review]
As far as I can judge, the changes are okay. However I'm not quite good in English grammar.

File: changelogs/unreleased/gh-7129-improve-check-for-dangerous-select-calls.md (hunk 2 above)
Comment (xuniq) [reply changelogs/unreleased/gh-7129-improve-check-for-dangerous-select-calls.md]: Fixed

**Review by kyukhin — APPROVED** [review]

**Pull request merged**